lib/actions/device.coffee

```diff
@@ -293,6 +308,71 @@ exports.hasDeviceUrl =
 			console.log(hasDeviceUrl)
 		.nodeify(done)
 
+exports.getApplicationInfo =
+	signature: 'device application info <uuid>'
+	description: 'get application container information'
+	help: '''
+		Use this command to get application container information from device
+
+		Examples:
+
+			$ resin device application info 23c73a1
+	'''
+	permission: 'user'
+	action: (params, options, done) ->
+		resin = require('resin-sdk-preconfigured')
+		resin.models.device.getApplicationInfo(params.uuid).then (appInfo) ->
+			console.log(appInfo)
+		.nodeify(done)
+
+exports.startApplication =
+    signature: 'device start application <uuid>'
+    description: 'start application on device'
+    help: '''
+        Use this command to start application on device
+
+        Examples:
+
+            $ resin device start application 23c73a1
+    '''
+    permission: 'user'
+    action: (params, options, done) ->
+        resin = require('resin-sdk-preconfigured')
+        resin.models.device.startApplication(params.uuid).then (containerId) ->
+            console.log(containerId)
+        .nodeify(done)
+
+exports.stopApplication =
+    signature: 'device stop application <uuid>'
+    description: 'stop application on device'
+    help: '''
+        Use this command to stop application on device
+
+        Examples:
+
+            $ resin device stop application 23c73a1
+    '''
+    permission: 'user'
+    action: (params, options, done) ->
+        resin = require('resin-sdk-preconfigured')
+        resin.models.device.stopApplication(params.uuid).then (containerId) ->
+            console.log(containerId)
```

Comment:
Almost certainly want to drop the console.log here too. Any reason you'd want the container id of the app you just stopped?